README.rst

```diff
@@ -29,6 +29,20 @@ Dependencies
 * gcc/gfortran or Intel compilers (clang might not work out of the box), if compiling from source
 * ducc0_, healpy, Cython, astropy, numpy, scipy, matplotlib, pyyaml, h5py, Pillow (Python Image Library)
 
+On MacOS, and other systems with non-traditional environments, you should specify the following standard environment variables:
+
+* ``CC``: C compiler (example: ``gcc``)
+* ``CXX``: C++ compiler (example: ``g++``)
+* ``FC``: Fortran compiler (example: ``gfortran``)
+
+We recommend using ``gcc`` installed from Homebrew to access these compilers.
+
+You will also need to specify the environment variable ``DUCC0_NUM_THREADS``, the number
```

Comment:
Maybe not for this PR, but @amaurea @mreineck and the two of us should come back to this. It's not clear to me why the number of threads should be specified at compile-time, rather than runtime.

Reply:
No, specifying `DUCC0_NUM_THREADS` at compile time won't achieve anything. It should behave exactly in the same way as `OMP_NUM_THREADS`, i.e. it affects the executable at runtime ... I just decided to use another name since there is no OpenMP at all in the code.

Reply:
Oh, if it really crashes, then there is a bug in `ducc` somewhere ... I had assumed that the code would just run on a single CPU if `DUCC0_NUM_THREADS` is not found. I'll try to locate the reason for this.

Reply:
Do you get any error message from ducc when such a crash happens?

Reply:
They all give: ```E      RuntimeError: 
E      ./src/ducc0/infra/threading.cc: 406 (thread_pool *ducc0::detail_threading::get_active_pool()):
E      
E      Assertion failure
E      no thread pool active
```

Reply:
Output of `env`:

```__CFBundleIdentifier=com.apple.Terminal
TMPDIR=/var/folders/k1/fld2m5s54fd9nd8qy_wdtvf00000gp/T/
XPC_FLAGS=0x0
TERM=xterm-256color
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.A1bC3VOARU/Listeners
XPC_SERVICE_NAME=0
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=C0631CF7-9288-4BA4-A8ED-D94334DF3A54
SHELL=/bin/zsh
HOME=/Users/borrow-adm
LOGNAME=borrow-adm
USER=borrow-adm
PATH=/Users/borrow-adm/.pyenv/shims:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
SHLVL=1
PWD=/Users/borrow-adm/Documents/SimonsObs/pixell
OLDPWD=/Users/borrow-adm/Documents/SimonsObs/pixell/pixell
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_REPOSITORY=/opt/homebrew
MANPATH=/opt/homebrew/share/man::
INFOPATH=/opt/homebrew/share/info:
ZSH=/Users/borrow-adm/.oh-my-zsh
PAGER=less
LESS=-R
LSCOLORS=Gxfxcxdxbxegedabagacad
LS_COLORS=di=1;36:ln=35:so=32:pi=33:ex=31:bd=34;46:cd=34;43:su=30;41:sg=30;46:tw=30;42:ow=30;43
PYENV_ROOT=/Users/borrow-adm/.pyenv
PYENV_SHELL=zsh
LANG=en_US.UTF-8
_=/usr/bin/env
```